game_quiz.py

```diff
@@ -0,0 +1,56 @@
+"""
+Welcome to Simple python quiz game
+"""
+import sys
+
+class Question:
+    """ The class based implementation of python quiz game"""
+    def __init__(self, prompt, answer):
+        self.prompt = prompt
+        self.answer = answer
+    @classmethod
+    def greet(cls):
+        """Function Introduce user about game"""
+        print("Welcome to AskPython Quiz")
+
+    @classmethod
+    def interest(cls):
+        """Function knows the interest of the user"""
+        play = input("Are you ready to play the Quiz ?")
+
+        if play != "yes":
+            sys.exit()
+            
+question_prompts = [
```

Comment:
Can be implemented as a dict rather than list
```suggestion
question_prompts = {
    "1": "What is your Favourite programming language?",
    "2": "Do you follow any author on AskPython?",
    "3": "What is the name of your favourite website for learning Python?"
}
```